From dba41f3d3da33d6c812a338f63e66ea3ec42d8ea Mon Sep 17 00:00:00 2001
From: Will Dietz <will.dietz@sifive.com>
Date: Mon, 13 May 2024 12:00:36 -0500
Subject: [PATCH] cmake: circt install directory for CAPI

---
 cmake/modules/AddCIRCT.cmake | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/cmake/modules/AddCIRCT.cmake b/cmake/modules/AddCIRCT.cmake
index 93379caa8..9f23c1661 100644
--- a/cmake/modules/AddCIRCT.cmake
+++ b/cmake/modules/AddCIRCT.cmake
@@ -11,8 +11,9 @@ function(add_circt_interface interface)
 endfunction()
 
 function(add_circt_public_c_api_library name)
-  add_mlir_public_c_api_library(${ARGV})
+  add_mlir_public_c_api_library(${ARGV} DISABLE_INSTALL)
   add_dependencies(circt-capi ${name})
+  add_circt_library_install(${name})
 endfunction()
 
 # Additional parameters are forwarded to tablegen.
-- 
2.43.2

