From b32c003aaa88dfcfc2c6119e16fd28820de8ffd7 Mon Sep 17 00:00:00 2001
From: Will Dietz <will.dietz@sifive.com>
Date: Fri, 9 Jan 2026 14:37:21 -0600
Subject: [PATCH] CMake: include installation destination use cmake var

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index a6345c3e11..0ed1e6bd2b 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -656,7 +656,7 @@ if (CIRCT_INCLUDE_DOCS)
 endif()
 
 install(DIRECTORY include/circt include/circt-c
-  DESTINATION include
+  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/include
   COMPONENT circt-headers
   FILES_MATCHING
   PATTERN "*.def"
@@ -668,7 +668,7 @@ install(DIRECTORY include/circt include/circt-c
   )
 
 install(DIRECTORY ${CIRCT_INCLUDE_DIR}/circt ${CIRCT_INCLUDE_DIR}/circt-c
-  DESTINATION include
+  DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/include
   COMPONENT circt-headers
   FILES_MATCHING
   PATTERN "*.def"
-- 
2.44.1

