From cd33530b148732fd0e9dcaf0d071493c9a08d718 Mon Sep 17 00:00:00 2001
From: Will Dietz <w@wdtz.org>
Date: Thu, 10 Mar 2022 15:38:18 -0600
Subject: [PATCH] lib install dir location

---
 cmake/modules/AddMLIR.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/modules/AddMLIR.cmake b/cmake/modules/AddMLIR.cmake
index 3c1ba3d8211f..8c770f8f2d44 100644
--- a/cmake/modules/AddMLIR.cmake
+++ b/cmake/modules/AddMLIR.cmake
@@ -363,8 +363,8 @@ function(add_mlir_library_install name)
   install(TARGETS ${name}
     COMPONENT ${name}
     ${export_to_mlirtargets}
-    LIBRARY DESTINATION lib${LLVM_LIBDIR_SUFFIX}
-    ARCHIVE DESTINATION lib${LLVM_LIBDIR_SUFFIX}
+    LIBRARY DESTINATION "${CMAKE_INSTALL_LIBDIR}${LLVM_LIBDIR_SUFFIX}"
+    ARCHIVE DESTINATION "${CMAKE_INSTALL_LIBDIR}${LLVM_LIBDIR_SUFFIX}"
     RUNTIME DESTINATION "${CMAKE_INSTALL_BINDIR}"
     # Note that CMake will create a directory like:
     #   objects-${CMAKE_BUILD_TYPE}/obj.LibName
-- 
2.35.1

