From adc7294a31ed4772b3d2755a7cee7dac90705326 Mon Sep 17 00:00:00 2001
From: Will Dietz <w@wdtz.org>
Date: Thu, 7 Aug 2025 12:53:50 -0500
Subject: [PATCH] hash.h: always use single-header boost_unordered

---
 include/slang/util/Hash.h | 9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

diff --git a/include/slang/util/Hash.h b/include/slang/util/Hash.h
index 749cd08b..dacff5e2 100644
--- a/include/slang/util/Hash.h
+++ b/include/slang/util/Hash.h
@@ -10,14 +10,7 @@
 #include <array>
 #include <cstring>
 
-#ifdef SLANG_BOOST_SINGLE_HEADER
-#    include <boost_unordered.hpp>
-#else
-#    include <boost/unordered/unordered_flat_map.hpp>
-#    include <boost/unordered/unordered_flat_set.hpp>
-#    include <boost/unordered/unordered_node_map.hpp>
-#    include <boost/unordered/unordered_node_set.hpp>
-#endif
+#include <boost_unordered.hpp>
 
 #include "slang/util/Util.h"
 
-- 
2.44.1

