From 678e169c8b74ff53d557133ada8ea1dcddcf32be Mon Sep 17 00:00:00 2001
From: Will Dietz <will.dietz@sifive.com>
Date: Tue, 21 Mar 2023 11:53:15 -0500
Subject: [PATCH] Drop span.hpp from install, was removed in
 f663d45cb9764c921b3377c98a199d78625589c0 .

---
 external/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/external/CMakeLists.txt b/external/CMakeLists.txt
index b0208651..105c9b36 100644
--- a/external/CMakeLists.txt
+++ b/external/CMakeLists.txt
@@ -45,7 +45,6 @@ if(SLANG_INCLUDE_INSTALL)
     COMPONENT slang_Development)
   install(
     FILES ${PROJECT_SOURCE_DIR}/external/flat_hash_map.hpp
-          ${PROJECT_SOURCE_DIR}/external/span.hpp
     DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}
     COMPONENT slang_Development)
   if(NOT unordered_dense_FOUND)
-- 
2.39.2

